src/styles/h5p-multi-media-choice.scss

```diff
@@ -161,4 +161,130 @@
     white-space: nowrap;
     width: 1px;
   }
+
+  .h5p-multi-media-content-media-button-video{
+    font-family: 'H5PFontAwesome4';
+    position: absolute;
+    background-color: crimson;
+    z-index: 3;
+    bottom: 0;
+    right: 0;
+    margin: 0.5em;
+    width: 3em;
+    height: 3em;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2),0 6px 20px 0 rgba(0,0,0,0.19);
+    cursor: pointer;
+    color: #ffffff;
+    border: 1px solid white;
+    border-radius: 1.5em;
+  }
+
+  /* Modal for video player */
+  .h5p-multi-media-content-media-video-modal {
+    position: absolute;
+    background-color:  rgba(0,0,0,0.8);
+    top:0;
+    left: 0;
+    padding: 0;
+    border: 1px solid #888;
+    z-index: 50;
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2),0 6px 20px 0 rgba(0,0,0,0.19);
+    animation-name: animatetop;
+    animation-duration: 0.4s;
+    display: none;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .h5p-multi-media-content-media-video-modal-container {
+    position: relative;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    max-width: 100%;
+    
+  }
+
+  .h5p-multi-media-content-media-video-modal-content {
+    position: relative;
+    background-color: white;
+    padding: 1em;
+    border: white;
+    width: 80%;
+    display: block;
+    max-width: 80%;
+    border-radius: 1%;
+    box-shadow: none;
+  }
+
+  .h5p-multi-media-choice-content {
+    position: relative;
+  }
+
+    /* modal Animation */
```

Comment:
No need for this comment, we know it since you declared the name in the correct place for the modal ˆˆ